packages/material-ui-icons/builder.js

```diff
@@ -85,64 +85,65 @@ function removeNoise(input, prevInput = null) {
   return removeNoise(output, input);
 }
 
-export async function cleanPaths({ svgPath, data }) {
+export function cleanPaths({ svgPath, data }) {
   // Remove hardcoded color fill before optimizing so that empty groups are removed
   const input = data
     .replace(/ fill="#010101"/g, '')
     .replace(/<rect fill="none" width="24" height="24"\/>/g, '')
     .replace(/<rect id="SVGID_1_" width="24" height="24"\/>/g, '');
 
-  const svgo = new SVGO({
+  const result = svgo.optimize(input, {
     floatPrecision: 4,
+    multipass: true,
     plugins: [
-      { cleanupAttrs: true },
-      { removeDoctype: true },
-      { removeXMLProcInst: true },
-      { removeComments: true },
-      { removeMetadata: true },
-      { removeTitle: true },
-      { removeDesc: true },
-      { removeUselessDefs: true },
-      { removeXMLNS: true },
-      { removeEditorsNSData: true },
-      { removeEmptyAttrs: true },
-      { removeHiddenElems: true },
-      { removeEmptyText: true },
-      { removeEmptyContainers: true },
-      { removeViewBox: true },
-      { cleanupEnableBackground: true },
-      { minifyStyles: true },
-      { convertStyleToAttrs: true },
-      { convertColors: true },
-      { convertPathData: true },
-      { convertTransform: true },
-      { removeUnknownsAndDefaults: true },
-      { removeNonInheritableGroupAttrs: true },
+      { name: 'cleanupAttrs' },
```

Comment:
Can be just strings `{ name: 'cleanupAttrs' }` -> ` 'cleanupAttrs'`

Reply:
sure, but can't be with params so let's optimize for change.

Reply:
@dependabot squash and merge